### PR TITLE
Update AEC and VAD features.

### PR DESCRIPTION
**Pull request by Juniverse**
If VAD is enabled. When player speaking detected, we'll pause the audio playing. So that the AEC quality would be improved.

Also added back the cache list for cancelled items.

INTG-1997, INTG-1980, INTG-1967, INTG-1908

### CODE REVIEW

File: Assets/Inworld/Inworld.Editor/Editor/InworldEditor.cs [hunk 1]
```diff
@@ -172,6 +173,24 @@ public InworldCharacter InnequinPrefab
...
+        {
+            get
+            {
+                if (InworldAI.User.Name.ToLower() != k_DefaultPlayerName)
+                    return InworldAI.User.Name;
+                return Instance.m_InputUserName;
+            }
+            set
+            {
+                if (Instance && value == Instance.m_InputUserName)
```
Comment (mattkwilson) [inline Assets/Inworld/Inworld.Editor/Editor/InworldEditor.cs:188]: I think you may have meant this to be: `!Instance || value == Instance.m_InputUserName`, but we could also just use `!Instance`

File: Assets/Inworld/Inworld.Samples.RPM/Prefabs/Character/Alice the Guide.prefab
Comment (mattkwilson) [inline Assets/Inworld/Inworld.Samples.RPM/Prefabs/Character/Alice the Guide.prefab:1]: Did you mean to add a Box Collider and Rigidbody?

**Review by mattkwilson — CHANGES_REQUESTED** [review]
AEC and VAD features appear to be working well. Please see comments and check the editor changes for the user name.

File: Assets/Inworld/Inworld.Editor/Editor/InworldEditor.cs (hunk 1 above)
Comment (Juniverse) [reply Assets/Inworld/Inworld.Editor/Editor/InworldEditor.cs:200]: No I added this check only for preventing redundant setting.

**Pull request closed without merging**